app/pages/api.py

```diff
@@ -38,7 +44,6 @@ def study_json(publicId):
         data.update({
             'description': study.description,
             'url':         study.url,
-            'timeUnits':   study.timeUnits,
```

Comment:
Returning time units here is a bit misleading, which is why I removed it. This field is really used to record what units the data was uploaded in, but it's always *returned* in hours, which is why it doesn't make sense to provide it in the API response, since the user can't do anything with it.